multidiff/Render.py

```diff
@@ -121,14 +138,41 @@ def _add_hex_space(self):
 			self.skipspace = False
 		else:
 			self.hexrow += ' '
-		
+
 
 	def final(self):
 		self.hexrow += 3*(16 - self.rowlen) * ' '
 		self.asciirow += (16 - self.rowlen) * ' '
 		self._newrow()
 		return self.body
 
+	def reformat(self, body, n=16):
```

Comment:
It doesn't need to be this complicated. Just find the number ```16``` from Render.py and make it not hardcoded. For example replace all occurencies of ```16``` with ```self.bytewidth``` and set that variable when constructing the ```Render``` object. This way there's no need to compile the regular expressions, find strings and do all the reformatting that takes a lot of time (you mentioned this was an issue?). As a side effect the format (addresses, ascii conversion etc.) will match the original

Reply:
Yes. I tried the approach you are suggesting first but then the code runs slowly. I don't know if it's a problem with my machine or not. Please tell me if by changing those lines the code runs the same in your machine. I would be happy to implement that since it's easier :)